packages/create-styles/src/createStyleSystem/tags.js

```diff
@@ -66,7 +67,6 @@ export const tags = [
 	'main',
 	'map',
 	'mark',
-	'marquee',
```

Comment:
`marquee` is deprecated and not supported by React. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L3052

Reply:
Ah, I vaguely remember this from emotion. Feel free to remove or comment out 👍